.github/component_owners.yml

```diff
@@ -64,6 +64,8 @@ components:
   propagators/opentelemetry-propagator-aws-xray:
     - NathanielRN
     - willarmiros
+  propagators/opentelemetry-propagator-instana:
+    - basti1302
```

Comment:
I wonder if we should also add `kirrg001`? WDYT? (Same for the owners declaration in the README.)

Reply:
Yeah sounds good 👍

Reply:
Done 👍